### DISCUSSION TITLE
v-touch-pan: wrong behavior or work as designed?

### DISCUSSION THREAD

**Discussion by vneznaikin** [quasarframework/quasar#13527] [General - Components / Directives / etc]
### What happened?

If you click on a button with a small movement that is in the "v-touch-pan" area, the click event **will not fire**. At the same time, outside of the "v-touch-pan" click with movement works well.

### What did you expect to happen?

I guess "v-touch-pan" blocks other events when triggered.
And I don't understand why it works the way it does. **What does not allow processing the events of the "v-touch-pan" area and the events of the button separately, independently of each other?**

### Reproduction URL

https://codepen.io/gradopetr/pen/GRyaRwW

### How to reproduce?

try in codepen

### Flavour

Quasar CLI (@quasar/cli | @quasar/app)

### Areas

Directives (quasar)

### Platforms/Browsers

Chrome

### Additional context

Yes @touchstart.stop and @mousedown.stop - can help in this situation, but it's kind of weird

**Comment by vneznaikin** [comment]
Big problem when element must be clickable and draggable.
@touchstart.stop and @mousedown.stop break 'draggable', without it click work bad

**Comment by pdanpdan** [comment]
I don't understand exactly your problem, but I suppose you want a move tolerant button
https://codepen.io/pdanpdan/pen/vYdNdEM?editors=1010

**Comment by vneznaikin** [comment]
Yes. It's works like a sharm!
I'm trying to make a drawer that can be opened not only by pulling the tab, but by clicking on it too. Sometimes this is handy for desktop users.
I was just trying to do something similar but you did it better. Thx!

However, given that my 'v-touch-pan' handles the entire screen, I have to put @touchstart.stop and @mousedown.stop on all other buttons. it's a little inconvenient. I don't know if this can be fixed in the future. Maybe there is some technical reason for that...

**Comment by pdanpdan** [comment]
You want something like this?
https://codepen.io/pdanpdan/pen/ExPqZbK

**Comment by vneznaikin** [comment]
Yes. But my drawers opening by dragging/touchmove anywhere on the display. In mobile style. But for desktop users, this may not seem obvious. For this, an additional reaction to clicking and dragging by the drawer tab.
And since  'v-touch-pan' takes up the entire screen area, all other buttons need to specify @touchstart.stop and @mousedown.stop

**Comment by pdanpdan** [comment]
Moved this to discussions as it's not a bug in implementation and it depends on user code implementation
